discord/ext/menus/pagination.py

```diff
@@ -0,0 +1,160 @@
+from discord.commands import ApplicationContext
+from discord.ext.commands import Context
+
+
+class Paginate(discord.ui.View):
+    """Creates a paginator for a message that is navigated with buttons.
+    
+    Parameters
+    ------------
+
+    pages: Union[List[:class:`str`], List[:class:`discord.Embed`]]
+        Your list of strings or embeds to paginate
+
+    show_disabled: :class:`bool`
+        Choose whether or not to show disabled buttons
+
+    """
+
+    def __init__(self, pages: Union[List[str], List[discord.Embed]], show_disabled = True, author_check = True):
+        super().__init__()
+        self.pages = pages
+        self.current_page = 1
+        self.page_count = len(self.pages)
+        self.show_disabled = show_disabled
+        self.forbutton = self.children[1]
+        self.prevbutton= self.children[0]
+        self.usercheck = author_check
+        self.user = None
+        if not self.show_disabled:
+            self.remove_item(self.children[0])
+            if self.page_count == 1:
+                self.remove_item(self.children[1])
+
+    async def interaction_check(self, interaction):
+        if self.usercheck:
+            return self.user == interaction.user
+        return True
+
+    @discord.ui.button(label = "<", style = discord.ButtonStyle.green, disabled=True)
+    async def previous(self, button: discord.ui.Button, interaction: discord.Interaction):
+
+        self.current_page -= 1
+
+        if self.current_page == 1:
+            button.disabled = True
+
+        if not self.show_disabled:
+            if len(self.children) == 1:
+                self.add_item(self.forbutton)
+                self.forbutton.disabled = False
+            if button.disabled:
+                self.remove_item(button)
+        else:
+            self.children[1].disabled = False
+
+        page = self.pages[self.current_page-1]
+        await interaction.response.edit_message(content = page if isinstance(page, str) else None, embed = page if isinstance(page, discord.Embed) else MISSING , view = self)
+
+    @discord.ui.button(label = '>', style=discord.ButtonStyle.green)
+    async def forward(self, button: discord.ui.Button, interaction: discord.Interaction): 
+
+        self.current_page += 1
+
+        if self.current_page == self.page_count:
+            button.disabled = True
+    
+        if not self.show_disabled: 
+            if len(self.children) == 1:
+                self.add_item(self.prevbutton)
+                self.prevbutton.disabled = False
+                self.children[0], self.children[1] = self.children[1], self.children[0]
+            if button.disabled:
+                self.remove_item(button)
+        else:
+            self.children[0].disabled = False
+
+        page = self.pages[self.current_page-1]
+        await interaction.response.edit_message(content = page if isinstance(page, str) else None, embed = page if isinstance(page, discord.Embed) else MISSING, view = self)
```

Comment:
```suggestion
        await interaction.response.edit_message(content = page if isinstance(page, str) else None, embed = page if isinstance(page, discord.Embed) else None, view = self)
```
With `MISSING`, the embed from a previous page isn't removed properly if the new page doesn't contain an embed. Replacing with `None` fixes this.